downstream/modules/platform/ref-general-inventory-variables.adoc

```diff
@@ -42,7 +37,24 @@ For more information, see xref:proc-set-registry-username-password[Setting regis
 If `routable_hostname` is not set, it should default to `ansible_host`. If you do not set `ansible_host`, `inventory_hostname` is used as a last resort.
 
 This variable is used as a host variable for particular hosts and not under the `[all:vars]` section. 
-For further information, see link:https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#assigning-a-variable-to-one-machine-host-variables[Assigning a variable to one machine:host variables].
+For further information, see link:https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#assigning-a-variable-to-one-machine-host-variables[Assigning a variable to one machine: host variables].
+
+|*`nginx_tls_protocols`* | Defines support for `ssl_protocols` in Nginx.
```

Comment:
Why is nginx_tls_protocols after routable_hostname? they should be in alphabetical order to make them easier to find.